### PR TITLE
feat: added test coverage for IHRC632 query functions

### LINKED ISSUE #1082: [HIP-632] Add test coverage for IHRC632 methods
### Description
The IHRC632.sol smart contract is already implemented in the repository with all the required methods. However, there is currently no test coverage for this contract. To ensure its functionality, we need to write comprehensive tests that validate the behavior of all implemented methods, including edge cases and failure scenarios.

### Solution
Add test coverage for IHRC632

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**:

- adds IHRC362 query functions to IHederaAccountService.sol and HederaAccountService.sol 
- adds test coverage for the query functions 

**Related issue(s)**:

Fixes #1082

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by andrewb1269hg — DISMISSED** [review]
File `.github/workflows/tests.yml` looks good.

File: test/system-contracts/hedera-account-service/ihrc-632/aliasAccountUtility.js [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+        EDItems[0].signerAlias.toLowerCase()
+      );
+      expect(incorrectSignerReceiptResponse[1].toLowerCase()).to.not.eq(
+        EDItems[1].signerAlias.toLowerCase()
+      );
+      expect(incorrectSignerReceiptResponse[2]).to.be.false;
+    });
+  });
+
+  describe(`IsAuthorized`, () => {
```
Comment (david-bakin-sl) [inline test/system-contracts/hedera-account-service/ihrc-632/aliasAccountUtility.js:298]: very nice test but could use a comment for the test as a whole: what's going on here

**Review by david-bakin-sl — APPROVED** [review]
LGTM (with some suggestions about comments, I think)

File: test/system-contracts/hedera-account-service/ihrc-632/aliasAccountUtility.js (hunk 1 above)
Comment (quiet-node) [reply test/system-contracts/hedera-account-service/ihrc-632/aliasAccountUtility.js:309]: you betcha. Just added a piece of comment on top of this suite. Please take a look and let me know if they are good enough.

**Review by andrewb1269hg — APPROVED** [review]

**Review by david-bakin-sl — APPROVED** [review]
LGTM
